src/status_im/android/core.cljs

```diff
@@ -32,14 +32,12 @@
                       "keyboardDidShow"
                       (fn [e]
                         (let [h (.. e -endCoordinates -height)]
-                          (dispatch [:hide-tab-bar])
```

Comment:
whos this related to assets ?

Reply:
Not related, but I found that these events are not used because the handling was moved into atom on tabbar component. Should I move it into a separate PR?

Reply:
https://github.com/status-im/status-react/pull/9843